database.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, Session
+
+SQLALCHEMY_DATABASE_ENGINE = "sqlite:///.library.db"
```

Comment:
The database URL `sqlite:///.library.db` seems to be missing a leading slash for the file path, which should be `sqlite:///./library.db` to correctly point to a file in the current directory. Ensure the path is correct to avoid connection issues.